### ISSUE TITLE
PHP sample source code

### ISSUE THREAD

**Issue by merdj** [gilek/bootstrap-gtreetable#16]
Hello,

your treetable is very nice, but the sample you provided which is downloadable is not enough due to the absence of the PHP file or on how to load it from JSON.

can you help by provided the files?

thank you so much

**Comment by lambdamusic** [comment]
+ 1 

I've tried adding an ajax method that returns this JSON, but still I see nothing on the page

```
{
  "level": "1",
  "type": "type",
  "id": "1",
  "name": "name"
}
```

**Comment by lambdamusic** [comment]
I kind of worked it out actually. The JSON needs to be like this:

```
	{"nodes":
		[
			{"id":4070,"name":"123.","level":0,"type":"default"},
			{"id":4100,"name":"asd","level":0,"type":"default"}
		]
	}	
```

The online demo for example works against this API:  http://gtreetable2.gilek.net/index.php?r=site%2FnodeChildren&id=0

**Comment by raghukotha** [comment]
B

**Comment by raghukotha** [comment]
Basically, you need to pass integers to both ID and Level fields. If you try to pass strings it will fail.

**Comment by robross0606** [comment]
Agreed, the example and documentation is really bad.  Almost impossible to determine how this is actually used.  What is the "template"?

**Comment by pipiscrew** [comment]
guy the example, using Yii2 extension aka[ https://github.com/gilek/yii2-gtreetable](url)
